README.md

```diff
@@ -391,19 +391,42 @@ rokka.stacks.get('myorg', 'mystack')
   .catch(function(result) {});
 ```
 
-#### rokka.stacks.create(organization, name, operations, [options=null], [overwrite=false]) → Promise
+#### rokka.stacks.create(organization, name, stackConfig, [params={}]) → Promise
 
 Create a new stack.
 
+The signature of this method changed in 0.27.
+
+The old signature still mostly works, but will be deprecated with 1.0.0.
+Not supported anymore since 0.27 is just using a single stack operation object (without wrapping it in an array)
+  as 3rd parameter
```

Comment:
`Using a single stack operation object (without an enclosing array) as the 3rd parameter (`stackConfig`) is since version 0.27.0 not supported anymore`

Not sure if it's better, your opinion?